src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3074,12 +3074,17 @@ public static void setMigrateKeycacheOnCompaction(boolean migrateCacheEntry)
 
     public static int getSSTablePreemptiveOpenIntervalInMiB()
     {
+        if (conf.sstable_preemptive_open_interval == null)
```

Comment:
I'd add some JavaDoc to the getter mentioning that it might return negative.

Reply:
This info is in the JavaDoc for the JMX method but I guess it doesn't hurt to add it here too.